### PR TITLE
Apple website Clone

### PR DESCRIPTION
**Pull request by KartheeKWD**
This project consists of building a webpage using images as a background and adding gradients to elements. The goal is to make a clone of an old version of the Apple website.

### CODE REVIEW

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+		</a>
+		<a class="verse" href="#">Watch the "Your Verse" film and explore the stories <img src="images/play-icon.ico" alt=""></a>
+	</main>
+	<section>
+		<a class="forward" href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone-5s/"></a>
+		<a class="colorful" href="https://web.archive.org/web/20140228163820/https://www.apple.com/iphone-5c/"></a>
+		<a class="birthday" href="https://web.archive.org/web/20140228125935/http://www.apple.com/30-years/"></a>
+		<a class="responsibility"
+			href="https://web.archive.org/web/20140301031333/http://www.apple.com/supplier-responsibility/"></a>
+	</section>
```
Comment (andresporras3423) [inline index.html]: You should move this section inside the main tag

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @KartheekWD @defoebrand** 
Amazing job with your first submission for this project :muscle:  
Your page really looks like the original :clap: :clap: :clap:  
Your project is almost perfect.There are just a couple of details to improve before you can go ahead:
- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload you ave 1 issue according to the validator.
- [ ] move the four images inside your main tag.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by peterrobert — APPROVED** [review]
### APPROVED :clap: :+1: :partying_face: 
**HELLO** :wave: **you have done such an impressive job**. Your readme file is descriptive. Your pull request looks amazing.  This is very impressive work :clap: 

You have also implemented the changes requested by the previous review :clap: 

**On to the next project**
### HAPPY CODING :computer:

**Pull request merged**